pagerduty/resource_pagerduty_event_orchestration_path_router.go

```diff
@@ -294,7 +364,12 @@ func expandRouterActions(v interface{}) *pagerduty.EventOrchestrationPathRuleAct
 	actions := new(pagerduty.EventOrchestrationPathRuleActions)
 	for _, ai := range v.([]interface{}) {
 		am := ai.(map[string]interface{})
-		actions.RouteTo = am["route_to"].(string)
+		dra := am["dynamic_route_to"]
+		if isNonEmptyList(dra) {
+			actions.DynamicRouteTo = expandRouterDynamicRouteToAction(dra)
+		} else {
+			actions.RouteTo = am["route_to"].(string)
+		}
```

Comment:
**Annotation:** Because `dynamic_route_to` and `route_to` are mutually-exclusive, we only set the `route_to` action if `dynamic_route_to` is not set